content/en/blog/2020/custom-istio-dns-name/index.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "Customizing Istio DNS stub names"
```

Comment:
the header (title, subtitle, description) I believe are not checked for spelling errors. Run `make lint` to verify.

Reply:
Im not sure what you're suggesting? Did it fail `make lint`?

Reply:
A bunch of words were added to the `.spelling`, possibly because you thought they would be caught up in the header spell checking. This is not needed. I will provide further guidance for the `.spelling` file above.

Reply:
I had noticed a bunch of words added to `.spelling`. I believe you may have taken this action to handle spelling errors in the header. I was just indicating the header is not spell-checked and requires extra attention from other reviewers for spelling errors.

Cheers
-steve